src/Scripts/Game/HUD/EL_BetaHud.c

```diff
@@ -0,0 +1,183 @@
+class EL_BetaHud : SCR_InfoDisplay
+{
+	private SliderWidget m_StaminaSlider;
+	private SliderWidget m_HealthSlider;
+	private SliderWidget m_ThirstSlider;
+	private SliderWidget m_HungerSlider;
+	private TextWidget m_MoneyDisplay;
+	private FrameWidget m_PlayerStatsFrame;
+	private FrameWidget m_PlayerStatsIcons;
+	private SCR_CharacterControllerComponent m_PlayerController;
+	private DamageManagerComponent m_DMC;
+	
+	private bool m_StatChange;
+	private float m_PreviousValues[4];
+	//TODO: Money & Survival Stats
+	
+	//--------------------------- OnChangeFunctions --------------------------- 
+	void OnHealthChange(float value)
+	{
+		if (!m_HealthSlider)
+		{
+			m_HealthSlider = SliderWidget.Cast(m_PlayerStatsFrame.FindWidget("healthSlider"));
+			if (!m_HealthSlider) return;
+		};
```

Comment:
No `;` after closing if statements. Please check the rest of the code to fix this